pkg/utils/mageutils/release.go

```diff
@@ -44,6 +44,32 @@ func CreateLocalTag(tag string) (err error) {
 	return sh.RunV("git", "tag", "-s", tag, "-m", "release "+tag)
 }
 
+// CreateAndPushTag executes "git", "tag", "-s", tag, "-m", "release "+tag
+// and after executes "git", "push", "--tags"
```

Comment:
Can we change this doc to something like this?
```
CreateAndPushTag create and push a new given tag.
```

I don't think that is clear document a function with the code it executes.

Reply:
OK, but this methods will be different of the others, I can continue?

Reply:
Oh, I didn't notice that all docs have this pattern. Maybe would be good leave it like this for consistency, or maybe we can merge the both?

```
CreateAndPushTag create and push a new given tag executing 
"git tag -s tag -m release+tag" and "git push --tags"
```

Reply:
Okay :)